src/App.tsx

```diff
@@ -1,35 +1,68 @@
 import { useState } from "react";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
-import Registration from "./components/Registration";
-import Sign from "./components/Sign";
-import Panel from "./components/Panel";
+import Registration from "./pages/Registration";
+import Sign from "./pages/Sign";
 import EditPage from "./pages/EditPage";
 import PostEdit from "./pages/PostEdit";
-import Dialog from "./components/Dialog";
+import Users from "./pages/Users";
+import Posts from "./pages/Posts";
+import Dashboard from "./pages/Dashboard";
+
+interface User {
+  id: number;
+  fullname: string;
+  country: string;
+  number: string;
+  email: string;
+  gender: string;
+}
+
+interface Post {
+  id: number;
+  title: string;
+  description: string;
+  date: string;
+  user: string;
+}
+
 function App() {
   const [active, setActive] = useState(false);
+  const [confirmActive, setConfirmActive] = useState(false);
```

Comment:
nu este sens sa folosesti in App use state-ul asta.
proiectul trebuie sa fie asa construit ca in App sa fie doar setari generale si /sau routing